src/main.rs

```diff
@@ -1,18 +1,20 @@
-mod blogs;
-mod posts;
+use std::convert::AsRef;
+use std::error::Error;
+use std::fs::{self, File};
+use std::io::Write;
+use std::path::{Path, PathBuf};
 
-use crate::blogs::Blog;
-use crate::posts::Post;
 use chrono::Timelike;
 use handlebars::{handlebars_helper, Handlebars};
 use sass_rs::{compile_file, Options};
 use serde_derive::Serialize;
 use serde_json::json;
-use std::convert::AsRef;
-use std::error::Error;
-use std::fs::{self, File};
-use std::io::Write;
-use std::path::{Path, PathBuf};
```

Comment:
Can we avoid whatever reformatting or shuffling was done here? It's going to make it harder to review and accept this PR -- and it shouldn't be necessary?